docs/appeng/adrs/0006-web-development-framework-preference.md

```diff
@@ -0,0 +1,92 @@
+# 0006 - Preference for x as web development framework(s)
+
+**Status:** Draft
+
+**Date Accepted:** x/x/xxxx
+
+## Context
+
+Our partners look to us for our expertise and our opinions on a variety of topics, including the tools we use to build software together. As we continue to seek new business it is beneficial for us to have an organizational opinion on what web development framework to use (when we have a choice in the matter).
+
+In making a choice we're seeking to match up the tradeoffs that the tools provide with projected partnership needs. The technical ecosystems we operate in tend to change often, so this is a decision we should seek to re-evaluate at least annually. Some topics to consider in the decision include:
+
+- Accessibility for junior engineers.
+- Support for rapid prototyping.
+- Ability to use in 'enterprise' environments.
+- Batteries included vs. things we have to roll ourselves.
+  - e.g authentication and authorization tooling, data model management, caching, etc
+
+(this list is likely not complete, feel free to modify or add to it as we put this draft together)
+
+The decision outcome does not need to be a single framework. Ideally we'll choose 2-3 that we would ask folks to choose from in most scenarios.
+
+## Decision
+
+Put a decision outcome here when ready.
+
+## Why is this applicable to the Practice as a whole?
+
+A large portion of our work is focused on developing web apps in partnership with public and private organizations. Each time something new starts we need to make decisions about how we'd like to approach it. This is meant to help give guidance in order to lower the burden of that choice.
+
+Having a 'practice opinion' on these tools will also help support folks' professional development, as well as our hiring and on-boarding processes. As we hire and get folks up to speed, this decision may be able to help us tailor what we do toward 'what we're trying to be good at.' Similarly we could lean on this as we seek to provide resources for folks developing their professional development plans.
+
+## Alternatives Considered
+
+### Django
+
+#### Pros
```

Comment:
Some more pros:
- Strong community support
- Built-in admin interface is really great for having a UI to interact with your database.

Reply:
Added these in!